src/test/java/liquibase/ext/databricks/AdvancedExtensionHarnessTestSuite.java

```diff
@@ -1,9 +1,10 @@
 package liquibase.ext.databricks;
 
 import liquibase.harness.AdvancedHarnessSuite;
+import liquibase.harness.generateChangelog.GenerateChangelogTest;
 import liquibase.harness.snapshot.SnapshotObjectTests;
 import org.junit.platform.suite.api.SelectClasses;
 
-@SelectClasses({SnapshotObjectTests.class})
```

Comment:
we can't enable this test until new version of test-harness with your fix is released. Otherwise, all our build will start failing